.github/workflows/build-assets.yml

```diff
@@ -330,8 +330,8 @@ jobs:
           build-args: |
             PKG_FILE=${{ steps.ln-container.outputs.k8s_pkg_file }}
           tags: |
-            ${{ secrets.JFROG_REGISTRY || 'dummy.io' }}/ln/platform-core-${{ matrix.os }}:${{ needs.preamble.outputs.internal_tag }}
-            ${{ secrets.JFROG_REGISTRY || 'dummy.io' }}/ln/platform-core-${{ matrix.os }}:${{ needs.preamble.outputs.candidate_base_branch }}
+            ${{ secrets.JFROG_REGISTRY || 'dummy.io' }}/hpccpl-docker-local/hpcc-platform/${{ matrix.os }}/platform-core-ln:${{ needs.preamble.outputs.internal_tag }} 
+            ${{ secrets.JFROG_REGISTRY || 'dummy.io' }}/hpccpl-docker-local/hpcc-platform/${{ matrix.os }}/platform-build-ln:${{ needs.preamble.outputs.candidate_base_branch }}
```

Comment:
I don't think line this is correct (or needed?)- this is not a 'build' image.

Reply:
That's a good point, with this new way there isn't a 'build-image'. We can likely delete line 334.

We're building the k8's package and then just importing it on top of an ubuntu-22.04 image that we install some prerequisite packages on top of.  That's the entire image.

Reply:
yes, I think is should be removed.